sdcm/argus_results.py

```diff
@@ -42,6 +42,9 @@ class Meta:
             ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION),
             ColumnMetadata(name="Overview", unit="", type=ResultType.TEXT),
             ColumnMetadata(name="QA dashboard", unit="", type=ResultType.TEXT),
+            ColumnMetadata(name="Throughput write", unit="ops", type=ResultType.INTEGER),
+            ColumnMetadata(name="Throughput read", unit="ops", type=ResultType.INTEGER),
+            ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION)
```

Comment:
Better put it before screenshot columns - will look better in Argus

Reply:
I think it is better to report throughput with latency, not separate it.

Reply:
I think the throughput stats should be placed with latency